mappings/net/minecraft/client/render/entity/model/ParrotEntityModel.mapping

```diff
@@ -1,2 +1,18 @@
 CLASS net/minecraft/class_584 net/minecraft/client/render/entity/model/ParrotEntityModel
+	FIELD field_3450 leftLeg Lnet/minecraft/class_630;
+	FIELD field_3451 innerBeak Lnet/minecraft/class_630;
+	FIELD field_3452 head Lnet/minecraft/class_630;
+	FIELD field_3453 outerBeak Lnet/minecraft/class_630;
+	FIELD field_3455 rightWing Lnet/minecraft/class_630;
+	FIELD field_3456 headFeathers Lnet/minecraft/class_630;
+	FIELD field_3457 rightLeg Lnet/minecraft/class_630;
+	FIELD field_3458 torso Lnet/minecraft/class_630;
+	FIELD field_3459 leftWing Lnet/minecraft/class_630;
+	FIELD field_3460 tail Lnet/minecraft/class_630;
+	FIELD field_3461 forehead Lnet/minecraft/class_630;
+	METHOD method_17105 poseOnShoulder (F)V
+	METHOD method_17106 animateModel (FFFFFI)V
```

Comment:
Inverted methods

Reply:
What do you mean by this?
It seems pretty clear to me at least that method_17106 is animateModel, as an extension of a more complex method.
I believe I copied them both from the newer version.

Reply:
In 1.16.3 `method_17106` is `poseOnShoulder`